src/main/java/core/basesyntax/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```

Comment:
Do we need to create this class?

Reply:
For task no, but we can see table structure with it for educational purposes